src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Img"
+        />
+      </a>
+
+      <nav>
+        <ul class="ul__section">
+          <li class="nav__section">
+            <a
+              href="#"
+              class="nav__button is-active first"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__section">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__section">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__section">
+            <a
+              href="#"
+              data-qa="hover"
+              class="nav__button"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__section">
+            <a
+              href="#"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. If this is a placeholder, consider using `href="#gadgets"` or another appropriate placeholder.